core/api/src/app/wallets/add-invoice-for-wallet.ts

```diff
@@ -262,8 +261,7 @@ const addInvoice = async ({
   const walletInvoiceBuilder = WalletInvoiceBuilder({
     dealerBtcFromUsd: dealer.getSatsFromCentsForFutureBuy,
     dealerUsdFromBtc: dealer.getCentsFromSatsForFutureBuy,
-    lnRegisterInvoice: (args) =>
-      lndService.registerInvoice({ ...args, sats: toSats(args.btcPaymentAmount.amount) }),
+    lnRegisterInvoice: lndService.registerInvoice,
```

Comment:
Weren't we using sats?

Reply:
We had a new interface that uses `btcPaymentAmount` instead that we hadn't switched over to, so I did the switchover here as well https://github.com/GaloyMoney/galoy/pull/4135/commits/cfd5055875b53eaa0d85ce65f79ea26bb8b3da8a